src/BlockTypes/AbstractProductGrid.php

```diff
@@ -647,7 +647,7 @@ protected function get_add_to_cart( $product ) {
 			'data-product_id'  => $product->get_id(),
 			'data-product_sku' => $product->get_sku(),
 			'rel'              => 'nofollow',
-			'class'            => 'wp-block-button__link add_to_cart_button',
+			'class'            => 'wp-block-button__link wp-element-button add_to_cart_button',
```

Comment:
Ideally, we should use `wp_theme_get_element_class_name( 'button' )` to get the correct class name, instead of directly using `wp-element-button`. Similar to what WC core does here: https://github.com/woocommerce/woocommerce/pull/35306/files#diff-3582a823c079f0c8354f3171ce608961ae6b224dd0182f7e4bb14e82278b05abR527-R543.

I don't know if there is an equivalent one for the JS side, but might be good investigating as well.

Reply:
Cool! Thanks for sharing this! I used `wc_wp_theme_get_element_class_name` to be more consistent with WC core [a00e4e6](https://github.com/woocommerce/woocommerce-blocks/pull/7516/commits/a00e4e60311b4fa407787a9451c55aab820f4537)